Traceback (most recent call last):
  File "main.py", line 67, in <module>
    param_iter, optimizer_text, lr_text, epoch_text = train_heat_equation_model_2d(init, Xu_noise, Xu_fixed,
  File "/local/232610/include/train.py", line 25, in train_heat_equation_model_2d
    init_loss = loss_fn(param_iter)
  File "/local/232610/include/heat2d.py", line 136, in heat_equation_nlml_loss_2d
    nlml = (1 / 2 * signed_logdet) + (1 / 2 * Y.T @ K_inv_Y.item()) + ((number_Y / 2) * jnp.log(2 * jnp.pi))
  File "/home/yanw/.conda/envs/uncertain/lib/python3.8/site-packages/jax/_src/numpy/array_methods.py", line 81, in _item
    return float(a)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.8/site-packages/jax/_src/array.py", line 263, in __float__
    return self._value.__float__()
TypeError: only size-1 arrays can be converted to Python scalars
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[25292,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: cannot stat '*.pdf': No such file or directory
cp: missing destination file operand after '*.png/home/yanw/uncertain/results/figures/0831'
Try 'cp --help' for more information.
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
